environments.json

```diff
@@ -1612,6 +1612,21 @@
       "esintl"
     ]
   },
+  "node8": {
+    "full": "Node.js",
+    "family": "Node.js",
+    "short": "Node &gt;=8 &lt;8.1.2",
```

Comment:
Why not just `Node 8`? Entries like that go out of date pretty quickly (in fact, it's already out of date as Node 8.1.3 is out). Upper bounds are needed once we know a new version introduces a v8 bump.